flattener/flatten.cc

```diff
@@ -14,9 +14,6 @@ namespace sorbet::flatten {
 // return true if the expression in question is either a method definition, a class definition, or an assignment to a
 // constant; false otherwise.
 bool isDefinition(core::Context ctx, const unique_ptr<ast::Expression> &what) {
-    if (ast::isa_tree<ast::MethodDef>(what.get())) {
```

Comment:
the comment above says that this should return true for methods.
Should the method name & comment be updated?

Reply:
Ah yeah forgot to cherry-pick that change back to this branch. It's coming in a stacked PR.